doc/command_ref.rst

```diff
@@ -183,6 +187,10 @@ Options
     Error output level. This is an integer value between 0 (no error output) and
     10 (shows all error messages), default is 3. Deprecated, use ``-v`` instead.
 
+``--enable, --set-enabled``
+    Enable specified repositories (automatically saves). The option has to by used together with
+    ``config-manager`` command (dnf-plugins-core).
```

Comment:
What about to create a link to dnf-plugins-core pages? (not important, just say no and I am fine with that)